sdks/python/apache_beam/runners/common.py

```diff
@@ -751,12 +763,25 @@ def _invoke_bundle_method(self, bundle_method):
     except BaseException as exn:
       self._reraise_augmented(exn)
 
+  def _invoke_lifecycle_method(self, lifecycle_method):
+    try:
+      self.context.set_element(None)
```

Comment:
Defensive programming to make sure the `lifecycle_method` does not abusively access `self.context.element`?

Reply:
Yes, also to maintain consistency with `_invoke_bundle_method`.